embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/AbstractJdbcInputPlugin.java

```diff
@@ -224,12 +227,9 @@ public ConfigDiff guess(ConfigSource config)
     protected ConfigDiff buildNextConfigDiff(PluginTask task, List<TaskReport> reports)
     {
         ConfigDiff next = Exec.newConfigDiff();
-        // TODO
-        //if (task.getOrderBy().isPresent()) {
-        //    // TODO when parallel execution is implemented, calculate the max last_value
-        //    //      from the all commit reports.
-        //    next.set("last_value", reports.get(0).get(JsonNode.class, "last_value"));
-        //}
+        if (reports.size() > 0 && reports.get(0).has("last_record")) {
+            next.set("last_record", reports.get(0).get(Map.class, "last_record"));
+        }
```

Comment:
last_record is lost if this bulk data loading loaded 0 records (that may happen as a successful case).